src/main/resources/assets/thaumcraftfix/recipes/quartz_sliver_to_block.json

```diff
@@ -0,0 +1,17 @@
+{
+  "type": "forge:ore_shaped",
+  "pattern": [
+    "QQQ",
+    "QQQ",
+    "QQQ"
+  ],
+  "key": {
+    "Q": {
+      "item": "thaumcraft:nugget",
```

Comment:
Thaumcraft registers quartz slivers as `nuggetQuartz` in the ore dictionary, so using that here would be better than hardcoding the item